src/targets/gpu/device/unary_not.cpp

```diff
@@ -7,13 +7,9 @@ inline namespace MIGRAPHX_INLINE_NS {
 namespace gpu {
 namespace device {
 
-void unary_not(hipStream_t stream,
-                 const argument& result,
-                 const argument& arg
-                 )
+void unary_not(hipStream_t stream, const argument& result, const argument& arg)
```

Comment:
these files were changed after a manual merge, but I believe the formatting is correct since it passes jenkins